pages/managing_relationships.md

```diff
@@ -125,7 +124,15 @@ And then the `Car` entity, as in the previous example:
     ? When you display this relationship with AngularJS, which field from 'Owner' do you want to use? id
 
 This will work as in the previous example, but you won't be able to add or remove cars from the `Owner` entity. On the generated AngularJS client UI you will have a dropdown in `Car` to select a `Owner`.
-This is not supported with JDL.
+This is the corresponding JDL:
```

Comment:
This has been changed a long time ago. I don't have time to review until tonight

Le 5 avril 2016 16:09:34 GMT+02:00, Deepu K Sasidharan <notifications@github.com> a écrit :
>> @@ -125,7 +124,15 @@ And then the `Car` entity, as in the previous
>example:
>>      ? When you display this relationship with AngularJS, which field
>from 'Owner' do you want to use? id
>>  
>>  This will work as in the previous example, but you won't be able to
>add or remove cars from the `Owner` entity. On the generated AngularJS
>client UI you will have a dropdown in `Car` to select a `Owner`.
>> -This is not supported with JDL.
>> +This is the corresponding JDL:
>
>@MathieuAA I thought you always create bi directional in JHUML? has
>this been changed in versions after 1.6.1?
>
>---
>You are receiving this because you were mentioned.
>Reply to this email directly or view it on GitHub:
>https://github.com/jhipster/jhipster.github.io/pull/236/files/d5b0966a49583481594fd6f7ab60eeeaf378721a#r58544106

-- 
Mathieu ABOU-AICHI